### PR TITLE
OpenVINO-EP v4.0 Release documentation update

### PR DESCRIPTION
**Pull request by MaajidKhan**
**Description**: Documentation update

**Motivation and Context**
- OpenVINO-EP now supports the latest OpenVINO 2022.1 version
- OpenVINO-EP is now updated to use the new OpenVINO 2.0 API's starting from this release

**source code PR:** https://github.com/microsoft/onnxruntime/pull/11025

### CODE REVIEW

File: docs/execution-providers/OpenVINO-ExecutionProvider.md [hunk 1]
```diff
@@ -36,6 +37,13 @@ Pre-built packages and Docker images are published for  ONNX Runtime with OpenVI
 
 For build instructions, please see the [BUILD page](../build/eps.md#openvino).
 
+**Announcement: OpenVINO-EP v4.0 Release with OpenVINO 2022.1 version**:
+* The new OpenVINO-EP v4.0 Release introduces a new version of OpenVINO API (API 2.0). For more information on the changes and transition steps, see the [transition guide](https://docs.openvino.ai/latest/openvino_2_0_transition_guide.html)
+
+* With this release, we have officialy stopped for centos. We will start supporting RHEL OS from
```
Comment (chausner) [inline docs/execution-providers/OpenVINO-ExecutionProvider.md:43]:
```suggestion
* With this release, we have officially stopped supporting CentOS. We will start supporting RHEL OS from
```

File: docs/execution-providers/OpenVINO-ExecutionProvider.md [hunk 2]
```diff
@@ -48,10 +56,9 @@ OpenVINO Execution Provider enables thread-safe deep learning inference
 
 ### Heterogeneous Execution for OpenVINO EP
 
-The heterogeneous Execution enables computing for inference on one network on several devices. Purposes to execute networks in heterogeneous mode
+The heterogeneous execution enables computing for inference on one network on several devices. Purposes to execute networks in heterogeneous mode:
 
-To utilize accelerators power and calculate heaviest parts of network on accelerator and execute not supported layers on fallback devices like CPU
-To utilize all available hardware more efficiently during one inference
+* To utilize accelerators power and calculate heaviest parts of network on accelerator and execute not supported layers on fallback devices like CPU to utilize all available hardware more efficiently during one inference.
```
Comment (chausner) [inline docs/execution-providers/OpenVINO-ExecutionProvider.md:61]:
```suggestion
* To utilize the accelerator's power and calculate the heaviest parts of the network on the accelerator and execute unsupported layers on fallback devices like the CPU to utilize all available hardware more efficiently during one inference.
```

File: docs/execution-providers/OpenVINO-ExecutionProvider.md [hunk 3]
```diff
@@ -73,28 +80,39 @@ Use `AUTO:<device 1><device 2>..` as the device name to delegate selection of an
...
+[Intel OpenVINO Auto Device Plugin](https://docs.openvino.ai/latest/openvino_docs_OV_UG_supported_plugins_AUTO.html).
 
 ### Model caching feature for OpenVINO EP
 
 The model caching setting enables blobs with Myriadx(VPU) and as cl_cache files with iGPU.
 
-#### Save/Load blob capability for Myriadx(VPU)
+### OpenCL queue throttling for GPU device
+
+Enables [OpenCL queue throttling](https://docs.openvino.ai/latest/groupov_runtime_ocl_gpu_prop_cpp_api.html?highlight=throttling) for GPU device. Reduces CPU Utilization when using GPU with OpenVINO EP.
```
Comment (chausner) [inline docs/execution-providers/OpenVINO-ExecutionProvider.md:91]:
```suggestion
### OpenCL queue throttling for GPU devices

Enables [OpenCL queue throttling](https://docs.openvino.ai/latest/groupov_runtime_ocl_gpu_prop_cpp_api.html?highlight=throttling) for GPU devices. Reduces CPU utilization when using GPUs with OpenVINO EP.
```

File: docs/execution-providers/OpenVINO-ExecutionProvider.md [hunk 4]
```diff
@@ -73,28 +80,39 @@ Use `AUTO:<device 1><device 2>..` as the device name to delegate selection of an
...
+
+#### Save/Load blob capability for Myriadx(VPU) with OpenVINO 2021.3 version
 This feature enables users to save and load the blobs directly. These pre-compiled blobs can be directly loaded on to the specific hardware device target and inferencing can be done. This feature is only supported on MyriadX(VPU) hardware device target.
 
-#### CL Cache capability for iGPU
+#### Model caching
 
-Starting from version 2021.4 OpenVINO supports [model caching](https://docs.openvino.ai/latest/openvino_docs_OV_UG_Model_caching_overview.html).
+Starting from version 2021.4 OpenVINO supports [model caching](https://docs.openvino.ai/latest/openvino_docs_OV_UG_Model_caching_overview.html). With OpenVINO 2021.4, it is supported on Myriadx(VPU)
```
Comment (chausner) [inline docs/execution-providers/OpenVINO-ExecutionProvider.md:98]:
```suggestion
Starting from version 2021.4, OpenVINO supports [model caching](https://docs.openvino.ai/latest/openvino_docs_OV_UG_Model_caching_overview.html). With OpenVINO 2021.4, it is supported on Myriadx(VPU)
```

File: docs/execution-providers/OpenVINO-ExecutionProvider.md [hunk 5]
```diff
@@ -73,28 +80,39 @@ Use `AUTO:<device 1><device 2>..` as the device name to delegate selection of an
...
+and iGPU.
 
-This feature enables users to save and load the cl_cache files directly. These cl_cache files can be directly loaded on to igpu hardware device target and inferencing can be done. This feature is only supported on iGPU hardware device target.
+From OpenVINO 2022.1 version, this feature is supported on Myriadx(VPU), iGPU and CPU.
+
+iGPU:
+This feature enables users to save and load the cl_cache files directly. These cl_cache files can be directly loaded on to igpu hardware device target and inferencing can be done.
```
Comment (chausner) [inline docs/execution-providers/OpenVINO-ExecutionProvider.md:104]:
```suggestion
This feature enables users to save and load the cl_cache files directly. These cl_cache files can be loaded directly onto the iGPU hardware device target and inferencing can be performed.
```

File: docs/execution-providers/OpenVINO-ExecutionProvider.md [hunk 6]
```diff
@@ -73,28 +80,39 @@ Use `AUTO:<device 1><device 2>..` as the device name to delegate selection of an
...
+
+Myriadx(VPU) and CPU:
+This feature enables users to save and load the blob file directly. This file can be directly loaded on to hardware device target and inferencing can be done.
```
Comment (chausner) [inline docs/execution-providers/OpenVINO-ExecutionProvider.md:107]:
```suggestion
This feature enables users to save and load the blob file directly. This file can be loaded directly onto the hardware device target and inferencing can be performed.
```

File: docs/execution-providers/OpenVINO-ExecutionProvider.md [hunk 7]
```diff
@@ -73,28 +80,39 @@ Use `AUTO:<device 1><device 2>..` as the device name to delegate selection of an
...
 
 There are two different methods of exercising this feature:
 
 #### <b>option 1. Enabling via Runtime options using c++/python API's.</b>
 
 This flow can be enabled by setting the runtime config option 'use_compiled_network' to True while using the c++/python API'S. This config option acts like a switch to on and off the feature.
 
-The blobs are saved and loaded from a directory named 'ov_compiled_blobs' from the executable path by default. This path however can be overridden using another runtime config option 'blob_dump_path' which is used to explicitly specify the path where you would like to dump and load the blobs (VPU) or cl_cache(iGPU) files from when already using the use_compiled_network(model caching) setting.
+The blobs are saved and loaded from a directory named 'ov_compiled_blobs' from the executable path by default. This path however can be overridden using another runtime config option 'blob_dump_path' which is used to explicitly specify the path where you would like to dump and load the blobs (VPU, CPU) or cl_cache(iGPU) files from when already using the use_compiled_network(model caching) setting.
```
Comment (chausner) [inline docs/execution-providers/OpenVINO-ExecutionProvider.md:115]:
```suggestion
The blobs are saved and loaded from a directory named 'ov_compiled_blobs' relative to the executable path by default. This path can be overridden using the runtime config option 'blob_dump_path' which is used to explicitly specify the path where you would like to dump and load the blobs (VPU, CPU) or cl_cache(iGPU) files from when already using the use_compiled_network(model caching) setting.
```

File: docs/execution-providers/OpenVINO-ExecutionProvider.md [hunk 8]
```diff
@@ -217,6 +236,7 @@ The following table lists all the available configuration options and the Key-Va
 | use_compiled_network | string | True/False | boolean | This option is only available for MYRIAD_FP16 VPU devices for both Linux and Windows and it enables save/load blob functionality. It can be used to directly import pre-compiled blobs if exists or dump a pre-compiled blob at the executable path. |
 | blob_dump_path | string | Any valid string path on the hardware target | string | Explicitly specify the path where you would like to dump and load the blobs for the save/load blob feature when use_compiled_network setting is enabled . This overrides the default path.|
 | context | string | OpenCL Context | void* | This option is only alvailable when OpenVINO EP is built with OpenCL flags enabled. It takes in the remote context i.e the cl_context address as a void pointer.|
+| enable_opencl_throttling | string | True/False | boolean | This option enables OpenCL queue throttling for GPU device (Reduces CPU Utilization when using GPU). |
```
Comment (chausner) [inline docs/execution-providers/OpenVINO-ExecutionProvider.md:239]:
```suggestion
| enable_opencl_throttling | string | True/False | boolean | This option enables OpenCL queue throttling for GPU devices (reduces CPU utilization when using GPU). |
```

**Comment by oliviajain** [comment]
https://oliviajain.github.io/onnxruntime/docs/execution-providers/OpenVINO-ExecutionProvider.html
https://oliviajain.github.io/onnxruntime/docs/build/eps.html

**Review by oliviajain — APPROVED** [review]

File: docs/execution-providers/OpenVINO-ExecutionProvider.md (hunk 1 above)
Comment (MaajidKhan) [reply docs/execution-providers/OpenVINO-ExecutionProvider.md]: Done

File: docs/execution-providers/OpenVINO-ExecutionProvider.md (hunk 2 above)
Comment (MaajidKhan) [reply docs/execution-providers/OpenVINO-ExecutionProvider.md]: Done

File: docs/execution-providers/OpenVINO-ExecutionProvider.md (hunk 3 above)
Comment (MaajidKhan) [reply docs/execution-providers/OpenVINO-ExecutionProvider.md]: Done

File: docs/execution-providers/OpenVINO-ExecutionProvider.md (hunk 4 above)
Comment (MaajidKhan) [reply docs/execution-providers/OpenVINO-ExecutionProvider.md]: Done

File: docs/execution-providers/OpenVINO-ExecutionProvider.md (hunk 5 above)
Comment (MaajidKhan) [reply docs/execution-providers/OpenVINO-ExecutionProvider.md]: Done

File: docs/execution-providers/OpenVINO-ExecutionProvider.md (hunk 6 above)
Comment (MaajidKhan) [reply docs/execution-providers/OpenVINO-ExecutionProvider.md]: Done

File: docs/execution-providers/OpenVINO-ExecutionProvider.md (hunk 7 above)
Comment (MaajidKhan) [reply docs/execution-providers/OpenVINO-ExecutionProvider.md]: Done

File: docs/execution-providers/OpenVINO-ExecutionProvider.md (hunk 8 above)
Comment (MaajidKhan) [reply docs/execution-providers/OpenVINO-ExecutionProvider.md]: Done

**Comment by MaajidKhan** [comment]
Thanks @chausner for the suggestions. I have added a commit to fix the typos and made all other relevant corrections as suggested.

**Comment by MaajidKhan** [comment]
Not sure why CheckLinks pipleine is failing. Looks like some links from other EP's and not OpenVINO.

**Review by jywu-msft — APPROVED** [review]

**Comment by jywu-msft** [comment]
> Not sure why CheckLinks pipleine is failing. Looks like some links from other EP's and not OpenVINO.

you need to merge the latest from the gh-pages branch to pick up https://github.com/microsoft/onnxruntime/commit/645ebcb5af34e5318ad49a65d864d13cd5ef8554

**Pull request merged**